modules/glnexus/meta.yml

```diff
@@ -0,0 +1,36 @@
+name: glnexus
+description: merge gVCF files and perform joint variant calling
+keywords:
+  - merge
+  - gvcf
+tools:
+  - glnexus:
+      description: scalable gVCF merging and joint variant calling for population sequencing projects.
+      homepage: https://github.com/dnanexus-rnd/GLnexus
+      documentation: https://github.com/dnanexus-rnd/GLnexus/wiki/Getting-Started
+      tool_dev_url: None
+      doi: https://doi.org/10.1101/343970
+      licence: ['Apache License 2.0']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - gvcfs:
+      type: list
+      description: Input genomic vcf files
+      pattern: "*.{g.vcf,g.vcf.gz}"
```

Comment:
```suggestion
      pattern: "*.{gvcf,gvcf.gz}"
```

Reply:
Are you suggesting this change to keep things standard across the pipelines? 

Also, where can I learn more about writing good meta.yml files? To write my module I followed the instructions [here](https://nf-co.re/developers/adding_modules#new-module-guidelines-and-pr-review-checklist), but it doesn't say much about meta files.

Reply:
Nope, I assumed the input files would either end in `.gvcf` or `.gvcf.gz`? Is that not right?

Reply:
Aah.. not always. For instance, [here](https://github.com/google/deepvariant/blob/r1.2/docs/deepvariant-quick-start.md) in deepvariant's quickstart example, the output gvcf file ends in g.vcf.gz. So I guess it really depends on the user. Do you think it is better to list all four suffixes here instead of picking two?

Reply:
Yep, in that case all four examples would be good 👍🏽 Thanks.

Reply:
Excellent! Will do!